csharp/Linked Lists/DoublyLinkedListNode.cs

```diff
@@ -0,0 +1,18 @@
+﻿public class DoublyLinkedListNode
```

Comment:
These classes are not needed in the repo since we just want to include the solution files in here